src/validators/employment.test.js

```diff
@@ -701,7 +701,7 @@ describe('Employment component validation', () => {
       {
         data: {
           EmploymentActivity: {
-            value: 'SelfEmployed',
+            value: 'StateGovernment',
```

Comment:
@christopherhuii Is there an opportunity to have a test that verifies that the additional question isn't answered/set e.g., for self-employment?